stable/chartmuseum/README.md

```diff
@@ -372,13 +372,56 @@ env:
     STORAGE_OPENSTACK_CONTAINER: mycontainer
     STORAGE_OPENSTACK_PREFIX:
     STORAGE_OPENSTACK_REGION: YOURREGION
+    # If your openstack API has a private/self-signed CA
+    # Put a path to a file here and dump you CA chain in ca_bundle below
+    STORAGE_OPENSTACK_CACERT: /etc/openstack.crt
+
   secret:
-    OS_AUTH_URL: https://myauth.url.com/v2.0/
-    OS_TENANT_ID: yourtenantid
+    OS_AUTH_URL: https://myauth.url.com/v3
+    OS_PROJECT_NAME: yourprojectname
+    OS_DOMAIN_NAME: yourdomainname
     OS_USERNAME: yourusername
     OS_PASSWORD: yourpassword
+openstack:
```

Comment:
please add these key descriptions to the table at the top of the readme

Reply:
Done